packaging_automation/upload_to_package_cloud.py

```diff
@@ -116,13 +116,16 @@ def package_exists(package_cloud_token: str, repo_owner: str, repo_name: str, pa
     parser.add_argument('--package_cloud_api_token', required=True)
     parser.add_argument('--repository_name', required=True, choices=supported_repos)
     parser.add_argument('--output_file_path', required=True)
+    parser.add_argument('--exec_folder_path', required=True)
+    parser.add_argument('--main_branch', required=True)
```

Comment:
I think we should document usage of those two new params in README file

Reply:
Already fixed in this PR